File: Core Java/player-app/src/com/techlabs/player/Player.java
package com.techlabs.player;

public class Player {

	private int id;
	private String name;
	private int age;

	public Player(int id, String name) {
		this(id, name, 18);
	}

	public Player(int id, String name, int age) {
		this.id = id;
		this.name = name;
		this.age = age;
	}

	public int getId() {
		return id;
	}

	public String getName() {
		return name;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}

	public Player whoIsElder(Player player) {
		if (this.age > player.age) {
			return this;
		}
		return player;
	}

	@Override
	public boolean equals(Object obj)
	{
		if(this.id==((Player) obj).getId())
		{
			return true;
		}
		else{
			return false;
		}
	}

	@Override
	public String toString() {
		return "Id: " + id + "\n" + "Name: " + name + "\n" + "Age: " + age
				+ "\n" +super.toString() +"\n";
	}
}

File: DOTNET/C#/PlayerApp/PlayerApp/Player.cs
using System;


namespace PlayerApp
{
    class Player
    {
        private readonly int _id;
        private readonly String _name;
        private int _age;

        public Player(int id, String name) : this(id, name, 18)
        {

        }

        public Player(int id, String name, int age)
        {
            _id = id;
            _name = name;
            _age = age;
        }

        public int Id
        {
            get
            {
                return _id;
            }
        }

        public String Name
        {
            get
            {
                return _name;
            }
        }

        public int Age
        {
            get
            {
                return _age;
            }
        }

        public Player WhoIsElder(Player player)
        {
            if (_age > player.Age)
            {
                return this;
            }
            return player;
        }

        public override bool Equals(object obj)
        {
            if (_id == ((Player)obj).Id)
            {
                return true;
            }
            else
            {
                return false;
            }

        }

        public override string ToString()
        {
            return "Id: " + _id + "\n" + "Name : " + _name + "\n" + "Age : " + _age + "\n";
        }

    }
}
